### PR TITLE
Updates for hc v0.0.47 with new zome templates

### PR DESCRIPTION
**Pull request by pdaoust**
Now that all the imports in the rust-proc zome template have been simplified, I've updated the tutorials to match the generated code. Please have a look over @freesig and make sure I haven't done anything stupid :D

### CODE REVIEW

**Comment by pdaoust** [comment]
@freesig is there any way I can force a CI on this branch?

**Comment by pdaoust** [comment]
dang, I can't seem to get the formatting right -- check out https://deploy-preview-211--fervent-tesla-2a3da2.netlify.com/docs/tutorials/coreconcepts/hello_me/

File: src/tutorials/coreconcepts/hello_gui.rs [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-extern crate serde_derive;
-extern crate serde_json;
-extern crate holochain_json_derive;
-use hdk::{
-    error::ZomeApiResult,
-};
 
+use hdk::prelude::*;
 use hdk_proc_macros::zome;
+
```
Comment (freesig) [inline src/tutorials/coreconcepts/hello_gui.rs:5]: Adding white space can break single source unfortunately. I'll probably need to pull this pr and test it

File: src/tutorials/coreconcepts/hello_holo.md [hunk 2]
```diff
@@ -128,41 +128,13 @@ The zome is a [Rust](https://rust-lang.com) project and makes use of [macros](ht
...
 Let's have a look at the generated code—--open up the `lib.rs` file in an editor.
 
-The following are all the imports. You are telling Rust, "Hey, I need things from all these [crates](https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html) in order to do my job."
+The following lines import the Holochain HDK. You are telling Rust, "Hey, I need things from all these [crates](https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html) in order to do my job."
 
-The `use` statements are next. They are saying, "I want to use these specific things from the above crates."
-You only need a few items for this tutorial, so go ahead and remove the others:
-\#S:CHANGE
+\#S:SKIP
 ```diff
```
Comment (freesig) [inline src/tutorials/coreconcepts/hello_holo.md:134]: Maybe this shouldn't be diff anymore?

File: src/tutorials/coreconcepts/hello_holo.md [hunk 3]
```diff
@@ -172,10 +144,10 @@ Remove the following piece of code:
 
 \#S:CHANGE
 ```diff
-- #[derive(Serialize, Deserialize, Debug, DefaultJson,Clone)]
-- pub struct MyEntry {
--     content: String,
-- }
+-#[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
```
Comment (freesig) [inline src/tutorials/coreconcepts/hello_holo.md:147]: Does the diff work without the space?

**Comment by freesig** [comment]
> @freesig is there any way I can force a CI on this branch?

Nope you should just move it into a develop branch and then compare to master

**Comment by freesig** [comment]
Ok this is fixed. It was the 
} that actually needed to be:
  }
for the check to work. Single source is actually pretty brittle. I'd love to rewrite it when we have time.
I think it would be cool to write it in clojure or something more suited to manipulating text.

**Pull request merged**